app/src/main/java/com/futurice/freesound/feature/search/SearchFragment.java

```diff
@@ -92,18 +96,18 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_search, container, false);
+        View view = inflater.inflate(R.layout.fragment_search, container, false);
```

Comment:
In the document they also use a `Binding.reset()` call in `onDestroyView()`. See http://jakewharton.github.io/butterknife/